Ugh similar problems to last time we bumped, on macos 10.15:

```
[ 91%] Linking C shared module mosquitto_dynamic_security.so
Undefined symbols for architecture x86_64:
  "_mosquitto_broker_publish", referenced from:
      _dynsec_control_callback in plugin.c.o
  "_mosquitto_callback_register", referenced from:
      _mosquitto_plugin_init in plugin.c.o
  "_mosquitto_callback_unregister", referenced from:
      _mosquitto_plugin_cleanup in plugin.c.o
  "_mosquitto_calloc", referenced from:
      _dynsec_auth__base64_decode in auth.c.o
      _dynsec_clients__config_load in clients.c.o
      _dynsec_clients__process_create in clients.c.o
      _dynsec_groups__config_load in groups.c.o
      _dynsec_groups__process_create in groups.c.o
      _mosquitto_plugin_init in plugin.c.o
      _dynsec_roles__config_load in roles.c.o
      ...
  "_mosquitto_client_id", referenced from:
      _dynsec_auth__basic_auth_callback in auth.c.o
      _dynsec_clients__process_create in clients.c.o
      _dynsec_clients__process_delete in clients.c.o
      _dynsec_clients__process_disable in clients.c.o
      _dynsec_clients__process_enable in clients.c.o
      _dynsec_clients__process_set_id in clients.c.o
      _dynsec_clients__process_set_password in clients.c.o
```

--- a/pkgs/servers/mqtt/mosquitto/default.nix
+++ b/pkgs/servers/mqtt/mosquitto/default.nix
@@ -17,13 +17,13 @@
 
 stdenv.mkDerivation rec {
   pname = "mosquitto";
-  version = "2.0.11";
+  version = "2.0.12";
 
   src = fetchFromGitHub {
     owner = "eclipse";
     repo = pname;
     rev = "v${version}";
-    sha256 = "1i0kkqhjxqahkhsl60s46zlywa87jakv7h4sr3fhi1v53vwpb9mg";
+    sha256 = "0bn6vpk6gdxrnm3aw3j2g0ny6cx2arv8pmv4x8302pr6qcrz57s6";
   };
 
   patches = lib.optionals stdenv.isDarwin [

